pkgs/development/interpreters/perl/default.nix

```diff
@@ -55,9 +56,8 @@ let
       ++ optional crossCompiling ./MakeMaker-cross.patch;
 
     postPatch = ''
-      pwd="$(type -P pwd)"
       substituteInPlace dist/PathTools/Cwd.pm \
-        --replace "/bin/pwd" "$pwd"
+        --replace "/bin/pwd" "${coreutils}/bin/pwd"
```

Comment:
Change to `$(type -P pwd`)`? That way you don't need an explicit dependency on coreutils.

Reply:
Oh wait, that's what it already did :-)